tests/schema/test_scalars.py

```diff
@@ -226,6 +227,146 @@ def echo_json_nullable(data: Optional[JSON]) -> Optional[JSON]:
     }
 
 
+def test_json_with_type_hints():
```

Comment:
Could you possibly put these files in another file (say, `tests/schema/test_json_types.py`)?

Reply:
Sure